src/main/cpp/locationinfo.cpp

```diff
@@ -102,6 +120,14 @@ const char* LocationInfo::getFileName() const
 	return fileName;
 }
 
+/**
+ *   Return the short file name of the caller.
+ *   @returns file name, may be null.
```

Comment:
`may be null` seems wrong to me when returning a non-pointer always. You mean `empty` instead?

Reply:
N